packages/components/src/tag/src/TagList.tsx

```diff
@@ -67,7 +67,7 @@ function TagItem({
                 }
             )}
         >
-            {item.content}
+            <>{item.content}</>
```

Comment:
Hmm, why?

Reply:
Same issue as in `getTextItem`. The type of `item.content` isn't `ReactNode`, but wrapping it in a fragment casts it into a `ReactNode`.

Reply:
I think I figured it out.

I changed the type of  `item.content` to be `ReactNode`, since it was only used to store `children` props.